README.md

```diff
@@ -1,16 +1,20 @@
 1. Replace `<your_account>` with your Github username in the link
     - [DEMO LINK](https://<your_account>.github.io/js_promise_function_DOM/)
 2. Follow [this instructions](https://mate-academy.github.io/layout_task-guideline/)
-    - There are no tests for this task so use `npm run lint` command instead of `npm test`
+    - Run `npm run test` command to test your code;
+    - Run `npm run test:only -- -n` to run fast test ignoring linter;
+    - Run `npm run test:only -- -l` to run fast test with additional info in console ignoring linter.
 
 ### Task: Promise function
 
 To complete this task, you should create a function that returns a `Promise`.
 Create function `waitFor` which takes two parameters:
-- `element` - DOM element
-- `eventName`
+- `element` - DOM element;
+- `eventName`, such as `click`, `input` and `blur`.
 
 The function returns a Promise which should be resolved when `eventName` happened on `element`.
 
-Hint: It doesn't matter in what way you choose to notify users about resolved or rejected promises. You can use DOM or `console.log`.
+Example of the promise: `It was click on the element: INPUT, id: login.`
```

Comment:
Not sure I understand this. Please rephrase or we can talk about this during the standup.

Reply:
changed description